charts/catalog/values.yaml

```diff
@@ -9,6 +9,7 @@ useAggregator: true
 ## If true, create & use RBAC resources
 ##
 rbacEnable: true
+rbacApiVersion: rbac.authorization.k8s.io/v1
```

Comment:
In **rbac.yaml** the `apiVersion` is set by `{{template "rbacApiVersion" . }}` because of that this variable will never be taken into account and it could be misleading, so what is the purpose of it? 

And also how this is solving the problem described in https://github.com/kubernetes-incubator/service-catalog/issues/2602?


I've executed locally such command _with helm client version v2.10.0_
`helm template --name=catalog --namespace=default  ./catalog`
and the `apiVersion` param was still empty.

Reply:
I agree, I don't think we can do this here. I agree that we should start defaulting to the v1 version.

Reply:
I fixed this

Reply:
You should also remove the `rbacApiVersion` definition from the **_helpers.tpl** file

Reply:
done! PTAL

Reply:
**I tested that and it's working. LGTM**
I was able to install the Service Catalog using this command
`helm template ./charts/catalog --name core --namespace default | kubectl apply -f -`

But one minor thing. I saw that the https://github.com/kubernetes-incubator/service-catalog/pull/2606 was merged. And those PRs are slightly overlapping. Here we using version from **values.yaml** file and in merged PR there was used the `{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" -}}`.

IMHO it could be unified. After merging this PR, for RBAC we will use the direct version from **values.yaml** and for APIService we will version based on `Capabilities`.


@MHBauer gives `approve` for https://github.com/kubernetes-incubator/service-catalog/pull/2606, so probably he has a broader view of the whole subject.